src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/InitializeStaticFieldsInlineTests.cs

```diff
@@ -155,6 +156,23 @@ End Class
 ");
         }
 
+        [Fact, WorkItem(3138, "https://github.com/dotnet/roslyn-analyzers/issues/3138")]
+        public async Task CA1810_EventSubscriptionInConstructor()
```

Comment:
I couldn't replicate the use case for vb.net (cannot access to `s` from the local lambda declaration).

Reply:
💭 

```
Public Class C
    Private Shared s As string
    
    Shared Sub New()
        AddHandler Console.CancelKeyPress,
            Sub(o, e)
                s = ""
            End Sub
    End Sub

End Class
```

Reply:
Stupid me, I missed the `Shared` on the field declaration...